src/qml/pages/onboarding/Onboarding00Welcome.qml

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The Onboarding00Welcome page.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+Pane {
+    background: Rectangle {
+        color: "black"
+    }
+
+    ColumnLayout {
+        anchors.centerIn: parent
+        spacing: 16
+
+        Image {
+            Layout.alignment: Qt.AlignCenter
+            source: "image://images/app"
```

Comment:
making a note that @bosch-0 has a proposal for a new icon. It needs to be discussed at greater length with the community; if there is consensus we should use it: https://github.com/bitcoin-core/gui/pull/199

Reply:
It's not just a new icon, there is also some changes to the watermark and thus overall logo. See this file for details: https://www.figma.com/file/c1V7b23n0LqRbVJlUkE1mn/Bitcoin-Core-App-Bosch?node-id=1850%3A44

Reply:
The new proposed icon is visually not centered in the circle. Can we address that, please?

Also, there are two oranges in the Figma file, Bitcoin Core orange and Bitcoin orange. They are slightly different, which one is the "right one"?

Reply:
I updated that awhile ago, looks visually centered to me?

![image](https://user-images.githubusercontent.com/55287964/138221699-019358a0-f595-46b5-949e-7bdf1774bf4b.png)


The Bitcoin Core orange is the primary orange, the Bitcoin orange is just there to illustrate the orange used by the Bitcoin logo - ill remove it from the stylesheet.